int8 MAXCHARACTERS = 20
string displayrow1
string displayrow2
string displayrow3
string displayrow4
